### ISSUE TITLE
Hide city banners / Friendly unit flags in city attack mode

### ISSUE THREAD

**Issue by chaorace** [CQUI-Org/cqui#450]
Holy moly it's annoying to try clicking around those things

**Comment by ricanuck** [comment]
Really?

**Comment by chaorace** [comment]
Okay, you disagree. What about this suggestion do you disagree with?

**Comment by ricanuck** [comment]
no, I really asked “Really?” because I never noticed a problem, range button is bigger and never interfered with anything... is there any set of conditions when it happens?

**Comment by UEAdmiral** [comment]
I agree, trying to click around flags and city banners when besieging a city or firing into a tight cluster of units has been an annoyance. Ranged attack just won't launch until I find a sweetspot that represents the location instead of an object, even if that object is present on that tile. 

I haven't installed this mod yet, but toggling show/hide or generally reducing unit flag size is definitely something I'd like to see.
